supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc

```diff
@@ -305,6 +338,51 @@ The API object for a deployment can be either a Kubernetes-native `Deployment` o
 ==== node (noun)
 See xref:node[node] in the _General Conventions_ part.
 
+[discrete]
+[[namespace store]]
+==== namespace store (noun)
+*Description*: A type of a resource for Multicloud Object Gateway that is used to store plain data and is used by Namespace buckets. The supported stores are S3, Azure, and S3 compatible.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[namespace bucket]]
+==== Namespace bucket (noun)
```

Comment:
Does "Namespace bucket" really capitalize the "Namespace", or can it be "namespace bucket"? I just spot checked the OCS docs and the instance I found used lowercase.